providers/africastalking/src/lib/africastalking.provider.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { AfricastalkingSmsProvider } from './africastalking.provider';
+
+test('should trigger africastalking library correctly', async () => {
```

Comment:
```suggestion
test(`should trigger Africa's Talking library correctly`, async () => {
```
Suggestion for the trickier situations due branding apostrophe.